antlr-kotlin-examples-jvm/src/test/kotlin/com/strumenta/antlrkotlin/examples/MiscStringTest.kt

```diff
@@ -11,19 +14,21 @@ fun assertArrayEquals(a: CharArray, b: CharArray) {
 class MiscStringTest {
 
 //    @Test fun testToCharArrayEmpty() {
-//        assertArrayEquals(charArrayOf(), "".toCharArray())
+//        com.strumenta.antlrkotlin.examples.assertArrayEquals(charArrayOf(), "".toCharArray())
```

Comment:
mmm, this seems weird

Reply:
I think this is a search&replace mistake